lib/rpc.ts

```diff
@@ -286,6 +301,14 @@ export class Rpc extends EventEmitter implements IForwarderDest {
     return this.postMessageForward(msg.mdest || "", msg.data);
   }
 
+  private _sendMessage(msg: IMessage): void {
```

Comment:
I think this should have the same signature as SendMessageCB type, and return the value of `this._sendMessageCB(msg)`

Reply:
done